src/main/java/org/broadinstitute/hellbender/engine/ReadWalker.java

```diff
@@ -29,6 +29,11 @@ public boolean requiresReads() {
     }
 
     /**
+     * This number dictates the number of lookeahead bases of the cache for the primary and the auxiliary features.
```

Comment:
lookeahead -> lookahead

Also, there are no "primary" sources of features for ReadWalkers, so re-word as:
"This number controls the size of the cache for our FeatureInputs (specifically, the number of additional bases worth of overlapping records to cache when querying feature sources)."